### ISSUE TITLE
[Feature Request] Serene Seasons / Tough as Nails support

### ISSUE THREAD

**Issue by EmergentCapulin** [progwml6/Natura#442]
TLDR - Would it be possible to provide support for the Serene Seasons / Tough as Nails mods?

This is not an issue, just something that I wanted to touch base about with your team. Glitchfiend recently released "Serene Seasons" and "Tough as Nails." Each of them have seasonal effects that alter how plants look in the gameworld in each season. As far as I've seen, and I haven't admittedly done a lot of probing, the Natura trees / plants don't change along with said seasons.

This is merely a hopeful request, would it be possible to add compatibility for those mods above? I'd love to see the full suite of Natura trees alter along with the seasons.  (Especially the Majestic Redwoods).

Thanks for your time, and the amazing work on the mod.

Capulin
